ci_framework/roles/openshift_setup/README.md

```diff
@@ -8,3 +8,4 @@ No privilege escalation needed.
 ## Parameters
 * `cifmw_openshift_setup_dry_run`: (Boolean) Skips resources creation. Defaults to `false`.
 * `cifmw_openshift_setup_create_namespaces`: (Strings) Namespaces to create beforehand. Defaults to `[]`.
+* `cifmw_openshift_setup_registry_tls_enable`: (Boolean) Enable tls_verify with openshift registy login. Defaults to `false`.
```

Comment:
why defaulting to false? we should really, by default, enable security.

Do you happen to have a node on hold with the failure? If it's a crc job, we're probably missing this:
```Bash
oc extract secret/router-ca --keys=tls.crt -n openshift-ingress-operator --confirm
sudo cp -f tls.crt /etc/pki/ca-trust/source/anchors/crc-router-ca.pem
sudo update-ca-trust
```

Reply:
@cjeanner this issue is random, We do not have node hold. I can try to hold the node with above issue.